flixel/input/mouse/FlxMouseEventManager.hx

```diff
@@ -6,28 +6,21 @@ import flixel.FlxBasic;
 import flixel.FlxCamera;
 import flixel.FlxG;
 import flixel.FlxObject;
-import flixel.FlxSprite;
 import flixel.group.FlxGroup.FlxTypedGroup;
 import flixel.input.mouse.FlxMouseButton.FlxMouseButtonID;
 import flixel.math.FlxPoint;
 import flixel.util.FlxDestroyUtil;
 
 /**
  * Provides mouse event detection for `FlxObject` and `FlxSprite` (pixel-perfect for those).
- * To use it, initialize the manager and register objects / sprites.
+ * The manager get initialized by `Flixel`. To use it, get the manager with 
+ * [`FlxG.plugins.get()`](https://api.haxeflixel.com/flixel/system/frontEnds/PluginFrontEnd.html#get) 
+ * and register objects / sprites:
```

Comment:
the static `FlxMouseEvent` class is meant to be the access point for the global manager, not via `FlxG.plugins`. It's Similar to `FlxTween` or `FlxTimer`. All of this doc should probably be moved there